codex-rs/execpolicy/tests/basic.rs

```diff
@@ -60,6 +62,49 @@ prefix_rule(
     );
 }
 
+#[test]
+fn add_prefix_rule_extends_policy() {
+    let mut policy = Policy::empty();
+    policy
+        .add_prefix_rule(&tokens(&["ls", "-l"]), Decision::Prompt)
+        .expect("add prefix rule");
+
+    let rules = rule_snapshots(policy.rules().get_vec("ls").expect("ls rules"));
```

Comment:
Coming up with messages for `expect()` in tests can be tedious. Consider changing the test (and others) to return `anyhow::Result<()>` so you can just use `?` instead. You can always leverage `anyhow::Context` if you want to add context to an error message.

The only real downside is the inevitable `Ok(())` at the end of the test.